src/thor/map_matcher.cc

```diff
@@ -121,17 +116,9 @@ interpolate_matches(const std::vector<meili::MatchResult>& matches,
   return interpolations;
 }
 
-// Form the path from the map-matching results. This path gets sent to
-// TripLegBuilder.
-std::vector<PathInfo>
-MapMatcher::FormPath(meili::MapMatcher* matcher,
-                     const std::vector<meili::MatchResult>& results,
-                     const std::vector<meili::EdgeSegment>& edge_segments,
-                     const std::shared_ptr<sif::DynamicCost>* mode_costing,
-                     const sif::TravelMode mode,
-                     std::vector<std::pair<GraphId, GraphId>>& disconnected_edges,
-                     Options& options) {
-
+uint32_t MapMatcher::compute_origin_epoch(const std::vector<meili::EdgeSegment>& edge_segments,
```

Comment:
nit: Flagging that we're using inconsistent capitalization. The default in meili seems to be capitalization: `MapMatcher::ComputeOriginEpoch`.

Reply:
these were not supposed to be moved out of the anonymous namespace in the source ill move them back